scripts/release.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
this script should take target comitish as parameters as well (commit candidates)

Reply:
Added in 0424292